### PR TITLE
Removing green background box on the logo

### LINKED ISSUE #3858: String showed when hoovering over title logo
**OS:** Win 7
**Version:** 0.0.5
**Commit/Build:** 304d797

- [ ] Reproducible in RCT2 (vanilla)?

Hovering over the icon on the titlescreen shows a string with the text 'Chairlift' which I presume works in all languages

This is caused by 
``	window->widgets = (rct_widget*)`` **0x009A9650**;
in src/windows/title_logo.c (around 'Create the window')

The hex are necessary for the game to not crash but also the culprit to make the string appare

bugs

### PR DESCRIPTION
**Pull request by Nubbie**
### Bug 1: Green background 
esgegs

**Optional:** Making the background fit the text, no idea why it's there tho

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Includes all git commit authors. Aliases are GitHub user names.
 * Michał Janiszewski (janisozaur) - Linux port, crash handling, security, misc.
 * Kelson Blakewood (spacek531) - title sequence for 0.0.4
 * Hugo Wallenburg (Goddesen) - Misc.
+* (Nubbie) - Misc, user friendlyness
```
Comment (IntelOrca) [inline contributors.md]: friendliness, although I'd just put UX

Comment (Nubbie) [inline contributors.md]: UX?

Comment (IntelOrca) [inline contributors.md]: Yes, but just write UX

File: src/windows/title_logo.c [hunk 2]
```diff
@@ -80,7 +80,7 @@ void window_title_logo_open()
 		WC_TITLE_LOGO,
 		WF_STICK_TO_BACK | WF_TRANSPARENT
 	);
-	window->widgets = (rct_widget*)0x009A9658; // mouse move bug in original game, keep this address and no crash happens
+	window->widgets = (rct_widget*)0x9A9638; // mouse move bug in original game, keep this address and no crash happens
```
Comment (IntelOrca) [inline src/windows/title_logo.c]: Why are you changing this, and also what to?

I will fix this afterwards, just just revert it for now.

Comment (Nubbie) [inline src/windows/title_logo.c]: By changing that value, it removes this completly
bugs

**Pull request merged**